### PR TITLE
#5919

### PR DESCRIPTION
**Pull request by yusharth**
## Description

**Fixes** #5919 

## First-time contribution

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by yusharth** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by martyna-mindsdb** [comment]
recheckcla

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 <div align="center">
 
+
```
Comment (martyna-mindsdb) [inline README.md:8]: Please remove the new line that you added here.

**Comment by martyna-mindsdb** [comment]
@yusharth 
Please see the review comment above and push the update to this PR.

**Pull request merged**